CircleciScripts/package_sdk.sh

```diff
@@ -74,6 +74,7 @@ mkdir -p $FRAMEWORK_DIR
 echo "copy framework"
 echo "builtFramework/Release-iphoneos/${project_name}.framework $FRAMEWORK_DIR"
```

Comment:
nit: remove, we should use `set -v` if we want to echo commands to console as we execute

Reply:
Updated.